synapse/config/_base.py

```diff
@@ -187,7 +282,8 @@ def generate_config(
         tls_private_key_path=None,
         acme_domain=None,
     ):
-        """Build a default configuration file
+        """
+        Build a default configuration file
```

Comment:
I kinda thought the `"""Foo` style was correct. I'd be interested if you have references either way.

Reply:
https://www.python.org/dev/peps/pep-0257/ says it can be either/or. With a new line is clearer IMO, since it provides some vertical space and means that every wrapped line in the docstring is the same length. (Twisted and Django have the `"""` on a separate line in their formal styleguides, and I see it in most modern Python libraries like attrs and such.)

Reply:
yeah the separate line is certainly easier to read. It'd be nice if we adopted some sort of convention on it, and I'm happy for it to be this.